Terminal.Gui/Text/TextFormatter.cs

```diff
@@ -579,7 +579,7 @@ public int GetHotKeySpecifierLength (bool isWidth = true)
     /// </remarks>
     public List<string> GetLines ()
     {
-        string text = _text!;
+        string text = _text!.ReplaceLineEndings ();
```

Comment:
Ha. This was one of the many things I had done in my old branch working on TextFormatter, in various places.

Reply:
I could argue this should be done on set_Text it doesn't happen each format.

Reply:
Yeah. That is where it is. There are/were a few things that are low hanging fruit like that.

Reply:
That's normalizing encoding for all applications. The `TestHelpers` class only do normalization on unit tests, that where it's needed. For applications where they are running on their own OS, then they are already normalized, unless if we open a file from another OS and thus the `ReplaceLineEndings` do the magic.